set.mm

```diff
@@ -481233,6 +481329,394 @@ such that every prime ideal contains a prime element (this is a
+        SYGUSYOYIGVQWJVSBDEGUARUHRYCWKXFTWLWSXAXGXDXIWSFXFWTWSXPFXFPXQABCDEFGLH
+        IJKWMWBZWNWSXCXHXBWSFXFOYPWOWPWQWR $.
+    $}
+
+    $d R i j k $.
+    $( The Zariski topology is a topology.  (Contributed by Thierry Arnoux,
+       16-Jun-2024.) $)
+    zartop $p |- ( R e. CRing -> J e. Top ) $=
+      ( vi vj vk ccrg wcel cprmidl cfv ctopon ctop clidl cv wss crab cmpt wceq
+      crn ccld eqid sseq1 rabbidv cbvmptv zartopn simpld topontop syl ) AIJZCAK
+      LZMLJZCNJUKUMFAOLZFPZGPZQZGULRZSZUACUBLTULABHGCUSDEULUCFHUNURHPZUPQZGULRU
+      OUTTUQVAGULUOUTUPUDUEUFUGUHULCUIUJ $.
+
+    ${
+      $d P i j k $.
+      zartop.3 $e |- P = ( PrmIdeal ` R ) $.
+      $( The base set of the Zariski topology.  (Contributed by Thierry Arnoux,
+         16-Jun-2024.) $)
+      zartopon $p |- ( R e. CRing -> J e. ( TopOn ` P ) ) $=
+        ( vi vj vk ccrg wcel ctopon cfv clidl cv wss crab cmpt wceq crn rabbidv
+        ccld sseq1 cbvmptv zartopn simpld ) BKLDAMNLHBONZHPZIPZQZIARZSZUADUCNTA
+        BCJIDUMEFGHJUHULJPZUJQZIARUIUNTUKUOIAUIUNUJUDUBUEUFUG $.
+    $}
+
+    $d J d x y $.  $d R d j k $.  $d R i j x y $.
+    $( The Zariski topology is T_0 .  (Contributed by Thierry Arnoux,
+       16-Jun-2024.) $)
+    zart0 $p |- ( R e. CRing -> J e. Kol2 ) $=
+      ( vx vd vy vj vi wcel cv cfv wss crab wceq wa sseq2 simpr cvv vk ccrg ct0
+      ctop wb clidl cprmidl cmpt crn wral wi zartop ssidd elrabd ad2antrr sseq1
+      rabbidv cbvmptv crg crngring simplr prmidlidl syl2anc rabex eqcomd adantl
+      fvex a1i elrnmptdv eleq2d bibi12d rspcdv imp mpbid elrab simprbi ad4ant13
+      syl mpbird eqssd anasss ralrimivva jca ctopon cuni eqid zartopon toponuni
+      ex ccld zartopn simprd ist0cld ) AUBKZCUCKCUDKZFLZGLZKZHLZWQKZUEZGUAAUFMZ
+      UALZILZNZIAUGMZOZUHZUIZUJZWPWSPZUKZHXFUJFXFUJZQWNWOXMABCDEULWNXLFHXFXFWNW
+      PXFKZWSXFKZXLWNXNQZXOQZXJXKXQXJQZWPWSXRWSWPXDNZIXFOZKZWPWSNZXRWPXTKZYAXPY
+      CXOXJXPXSWPWPNIWPXFXDWPWPRWNXNSXPWPUMUNUOXQXJYCYAUEZXQXAYDGXTXIXQJXBJLZXD
+      NZIXFOZWPXTXHTUAJXBXGYGXCYEPXEYFIXFXCYEXDUPUQURZXQAUSKZXNWPXBKWNYIXNXOAUT
+      UOZWNXNXOVAWPAVBVCXTTKXQXSIXFAUGVGZVDVHYEWPPZXTYGPXQYLYGXTYLYFXSIXFYEWPXD
+      UPUQVEVFVIXQWQXTPZQZWRYCWTYAYNWQXTWPXQYMSZVJYNWQXTWSYOVJVKVLVMVNYAXOYBXSY
+      BIWSXFXDWSWPRVOVPVRXRWPWSXDNZIXFOZKZWSWPNZXRYRWSYQKZWNXOYTXNXJWNXOQZYPWSW
+      SNIWSXFXDWSWSRWNXOSUUAWSUMUNVQXQXJYRYTUEZXQXAUUBGYQXIXQJXBYGWSYQXHTYHXQYI
+      XOWSXBKYJXPXOSWSAVBVCYQTKXQYPIXFYKVDVHYEWSPZYQYGPXQUUCYGYQUUCYFYPIXFYEWSX
+      DUPUQVEVFVIXQWQYQPZQZWRYRWTYTUUEWQYQWPXQUUDSZVJUUEWQYQWSUUFVJVKVLVMVSYRXN
+      YSYPYSIWPXFXDWPWSRVOVPVRVTWIWAWBWCWNFHXFXICGWNCXFWDMKZXFCWEPXFABCDEXFWFZW
+      GXFCWHVRWNUUGXICWJMPXFABJICXHDEUUHYHWKWLWMVS $.
+
+    ${
+      $d R i j k l $.  $d R k l m $.  $d T m $.
+      zarmxt1.1 $e |- M = ( MaxIdeal ` R ) $.
+      zarmxt1.2 $e |- T = ( J |`t M ) $.
+      $( The Zariski topology restricted to maximal ideals is T_1 .
+         (Contributed by Thierry Arnoux, 16-Jun-2024.) $)
```

Comment:
"T_1" = "Fre"? "T_1" = "Fréchet spaces" (see df-t1). Maybe this shoulkd be added here.

Reply:
Similar to previous comment, $T_1$ spaces are also called Fréchet spaces, and this is formalized as `Fre` in set.mm.
To me, the right place to explain this is indeed in [df-t1](https://us.metamath.org/mpeuni/df-t1.html). 

The prime spectrum restricted to maximal ideals is called the [maximal spectrum](https://ncatlab.org/nlab/show/maximal+spectrum) and denoted $mSpec$, $Spec_m$ or sometimes $Spec_{max}$. That would correspond to ``( S |`s M )``, and its topology would be the `` ( J |`t M ) `` of this theorem. But it does not have its own Wikipedia page. Maybe @savask can help tell if it's an important concept?

Reply:
I'm out of my depth here, but it seems that the concept is important enough (wikipedia does mention it in passing), but it doesn't behave that well as `Spec`.

Reply:
I'll leave it aside for now. 
From the Wikipedia page, it looks like it was Grothendieck's idea to generalize from the maximal spectrum $Spec_m$ to the prime spectrum $Spec$.
In our case we have all the tools to restrict, and many properties are inherited in that case. So it's worth noting that the maximal spectrum is easy to construct with the available formalization.